protocol/chat.go

```diff
@@ -40,6 +40,23 @@ const (
 	FirstMessageTimestampNoMessage = 1
 )
 
+const (
+	MuteFor15MinsDuration = 900
+	MuteFor1HrsDuration   = 3600
+	MuteFor8HrsDuration   = 28800
+	MuteFor1WeekDuration  = 604800
+	MuteFor1MinDuration   = 60
+)
+
+const (
+	MuteFor15MinsType = 1
```

Comment:
You can use `iota`  https://yourbasic.org/golang/iota/ , starting at 1, any specific reason why you want `MuteTill1MinType` as `99`?

Reply:
Because this was just a testing value to help QA test the functionality .. Will be removed once tested and approved